include/bluetooth/services/fast_pair.h

```diff
@@ -167,6 +179,16 @@ bool bt_fast_pair_has_account_key(void);
 int bt_fast_pair_battery_set(enum bt_fast_pair_battery_comp battery_comp,
 			     struct bt_fast_pair_battery battery);
 
+/** Register Fast Pair information callbacks.
+ *
+ *  This function supports registration of a single instance with information callbacks.
+ *
+ *  @param cb Callback struct.
```

Comment:
Consider informing that the structure needs to persist in memory after it's registered (to indicate that it's directly used later on without copying it).